Makefile

```diff
@@ -25,6 +25,8 @@ git_rev      = $(shell git rev-parse --short HEAD)
 git_tag      = $(subst v,,$(shell git describe --tags --abbrev=0))
 VERSION     ?= $(git_tag)
 
+dockerlogin = @echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
```

Comment:
@mdanzinger I had to make one more commit for the CI to pass. Just added a docker login to bypass the default rate limiting they put in place